Add MkrIoTCarrier datasheet (#156)
arduino/docs-content: content/hardware/01.mkr/03.carriers/mkr-iot-carrier/datasheet/datasheet.md

    * Low power consumption (2μA @ 1 Hz sampling rate)

* **Broadcom APDS-9660 RGB and Gesture Sensor**
    * Ambient light and RGB Colour Sensing
    * Proximity Sensing
    * Gesture Detection
    * UV/IR Blocking Filter
    * I2C interface to Arduino MKR board 

* **ST LPS22HBTR Pressure Sensor**
    * 260 to 1260 hPa (0.25 to 1.24 atm) absolute pressure range
    * I2C interface to Arduino MKR board 

* **Relays**
    * 2x KEMET EE2-5NU-L relays
    * 5V Coil voltage
    * 2A Current
    * 220V DC, 250V AC
    * Non-latching
    * Common, normally open and normally closed contacts
    * LED Status Indicator

* **Peripherals**
    * 5x Capacitive buttons
    * Buzzer
    * 5x Digital RGB LEDs
    * Rounded OLED 1,3” Display
    * 5x Capacitive qTouch buttons 

* **Memory**
    * Micro SD Card

* **Power**
    * Li-ion 18650 3.7 v battery holder
    * USB Battery charging via MKR Board (Runs up to 48h with a 3.7v 2500mAh)

* **I/O**
    * All sensors feature wake up function
    * 2x Analog Grove connectors
    * 1x I2C Grove connector
    * 2x relay connector

* **Safety information**
    * Class A 

# Contents

## The Board

### Application Examples

> **Light controller:** Control your house lightning with the IoT Carrier, using the RGB light sensor. This feature identifies the general amount of light in a room or environment and adapts. If the board is connected to WiFi,  you can manage and control your device remotely on a smartphone via the IoT Cloud.

Reviewer: ```suggestion
**Light controller:** Control your house lightning with the MKR IoT Carrier, using the RGB light sensor. This feature identifies the general amount of light in a room or environment and adapts. If the board is connected to Wi-Fi,  you can manage and control your device remotely on a smartphone via the IoT Cloud.
```

**Light and water controller for greenhouse:** With moisture sensor, pressure sensor and temperature the IoT Carrier can evaluate and recognize the moisture of the soil, depending on air humidity and sun. Through the various sensors, it can reconstruate suitable climate (e.g. tropical) with the help of heaters and relays to increase the humidity. It can also be programmed with a watering system thanks to the relays. 

**Weather station:** With temperature sensor, pressure sensor, humidity sensor and light sensors,  you can easily use your IoT carrier as a local weather station. The various sensors can via sensors collect statistics about the different variables required to display weather. 

### Accessories

* 18650 Li-ion battery
* Grove sensors 

### Related Products

> * Arduino MKR 1010 (SKU: ABX00023)

Reviewer: ```suggestion
* Arduino MKR WiFi 1010 (SKU: ABX00023)
```

* Protective case

> * Arduino MKR Zero

Reviewer: ```suggestion
* Arduino® MKR Zero
```

| U1       | LSM6DSOXTR 6-axis IMU IC               | U3       | LV52204MTTBG LED Boost Driver IC   |
| U2       | APDS-9660 RGB and Gesture Sensor IC    | HS-1     | HTS221 Humidity Sensor IC          |
| LPS-1    | LPS22HBTR Pressure Sensor IC           | L0-L4    | APA1022020-2018 RGB LED IC         |
| J6       | FH26W-45S-0.3SHW(60) Display Connector | J12      | SFV24R-1STBE1HLF Display Connector |

#### Back View

![Bottom view](assets/bottom_view.png)

| **Ref.**  | **Description**                       | **Ref.** | **Description**                   |
| --------- | ------------------------------------- | -------- | --------------------------------- |
| J2, J3    | Analog Grove Connectors               | J4       | I2C Grove Connector               |
| J11       | B2B-PH-SM4-TB(LF)(SN) Power Connector | J9, J10  | 1771033 3-pin Relay Connector     |
| JDIGITAL1 | MKR Power and Digital Pin Connector   | JANALOG1 | MKR AREF and Analog Pin Connector |
| S1, S2    | EE2-5NU-L 2A 250V Mechanical Relay    | M1       | Buzzer                            |
| J1        | Micro SD Module                       | J7, J8   | Li-Ion 18650 Holder               |

### Power Tree

> The Arduino MKR IoT Carrier makes use of the power management features of the Arduino MKR boards to power itself as well as to interface with the Li-Ion battery. 

Reviewer: ```suggestion
The MKR IoT Carrier makes use of the power management features of the Arduino MKR boards to power itself as well as to interface with the Li-Ion battery. 
```

![Power tree](assets/Power_Tree_MKR_IoT_Carrier.png)

### Solution Overview

![Solution overview](assets/solution_overview.png)

> The MKR IoT Carrier is also a part of the Oplá kit, that in addition to the IoT carrier contains an 
> 
> Arduino MKR WiFi 1010, a motion sensor, moisture sensor, USB cable and a protective case.  

Reviewer: ```suggestion
The MKR IoT Carrier is also a part of the Oplá kit, that in addition to the IoT carrier contains an Arduino MKR WiFi 1010, a motion sensor, moisture sensor, USB cable and a protective case.  
```

## Board Operation

### Getting Started - IDE 
If you want to program your Arduino MKR IoT carrier while offline you need to install the Arduino Desktop IDE **[1]**. To connect the Arduino IoT Carrier to your computer, you’ll need a Micro-B USB cable. This also provides power to the board, as indicated by the LED.

### Getting Started - Arduino Web Editor 
All Arduino boards, including this one, work out-of-the-box on the Arduino Web Editor **[2]**, by just installing a simple plugin.

The Arduino Web Editor is hosted online, therefore it will always be up-to-date with the latest features and support for all boards. Follow **[3]** to start coding on the browser and upload your sketches onto your board.

### Getting Started - Arduino IoT Cloud 
All Arduino IoT enabled products are supported on Arduino IoT Cloud which allows you to Log, graph and analyze sensor data, trigger events, and automate your home or business.

### Sample Sketches 

> Sample sketches for the Arduino IoT carrier can be found either in the “Examples” menu in the Arduino IDE or in the Opla Arduino website **[4]** and further documentation can be found in Arduino MKR **[8]** which also contains links to the troubleshooting articles.

Reviewer: ```suggestion
Sample sketches for the Arduino MKR IoT Carrier can be found either in the “Examples” menu in the Arduino IDE or in the Opla Arduino website **[4]** and further documentation can be found in Arduino MKR **[8]** which also contains links to the troubleshooting articles.
```

### Online Resources

Now that you have gone through the basics of what you can do with the board you can explore the endless possibilities it provides by checking exciting projects on ProjectHub **[5]**, the Arduino Library Reference **[6]** and the online store **[7]** where you will be able to complement your board with sensors, actuators and more

### Board Recovery 
All Arduino boards have a built-in bootloader which allows flashing the board via USB. In case a sketch locks up the processor and the board is not reachable anymore via USB it is possible to enter bootloader mode by double-tapping the reset button right after power up.

## Connector Pinouts

![Pinout](assets/Pinout.png)

> Detailed information on the IoT Carrier’s pinouts are available in a separate document: [Docs Arduino - MKR IoT Carrier pinout](https://docs.arduino.cc/resources/pinouts/ABX00047-full-pinout.pdf)

Reviewer: ```suggestion
Detailed information on the MKR IoT Carrier’s pinouts are available in a separate document: [Docs Arduino - MKR IoT Carrier pinout](https://docs.arduino.cc/resources/pinouts/ABX00047-full-pinout.pdf)
```

Hereby, Arduino S.r.l. declares that this product is in compliance with essential requirements and other relevant provisions of Directive 2014/53/EU. This product is allowed to be used in all EU member states. 

| Frequency bands | Maximum output power (ERP) |
| :-------------: | :------------------------: |
|   863-870Mhz    |          -3.22dBm          |

## Company Information

| Company name    | Arduino S.r.l.                             |
| --------------- | ------------------------------------------ |
| Company Address | Via Andrea Appiani,25 20900 MONZA (Italy） |

## Reference Documentation

| **Ref**                   | **Link**                                                     |
| ------------------------- | ------------------------------------------------------------ |
| Arduino IDE (Desktop)     | https://www.arduino.cc/en/Main/Software                      |
| Arduino IDE (Cloud)       | https://create.arduino.cc/editor                             |
| Cloud IDE Getting Started | https://create.arduino.cc/projecthub/Arduino_Genuino/getting-started-with-arduino-web-editor-4b3e4a |

> | Arduino Opla              | https://opla.arduino.cc/                                     |

Reviewer: ```suggestion
| Arduino® Oplà IoT Kit              | https://opla.arduino.cc/                                     |
```

| Project Hub               | https://create.arduino.cc/projecthub?by=part&part_id=11332&sort=trending |
| Library Reference         | https://www.arduino.cc/reference/en/libraries/arduino_mkriotcarrier/ |
| Online Store              | https://store.arduino.cc/mkr-iot-carrier                     |

> | Arduino MKR               | https://docs.arduino.cc/hardware/mkr-iot-carrier             |

Reviewer: ```suggestion
| Arduino Documentation               | https://docs.arduino.cc/hardware/mkr-iot-carrier             |
```
